scripts/conda.sh

```diff
@@ -0,0 +1,9 @@
+echo "Creating conda env..."
+conda create -n epispot python=3.9
```

Comment:
```suggestion
conda create -n epispot
```
Use the user-selected default Python version. After all, epispot supports Python 3.6-3.9, so we don't need to worry about this.